### PR TITLE
posthog: update instructions for Cloud EU

### PR DESCRIPTION
**Pull request by xvello**
### Proposed changes

- explicitly document the `PostHog instance` param, with the right value for EU users
- link to our community slack for help

### Merge timing

- ASAP once approved

### Related issues (optional)

Questions on our [community slack](https://posthogusers.slack.com/archives/C01GLBKHKQT/p1669804832343179)

### CODE REVIEW

File: src/connections/destinations/catalog/posthog/index.md [hunk 1]
```diff
@@ -16,9 +16,10 @@ This destination is maintained by PostHog. For any issues with the destination,
 3. Choose which Source should send data to the PostHog destination.
 4. Go to your [PostHog set up page](https://app.posthog.com/setup), and find and copy the **API key**.
 5. Enter the PostHog API Key that you copied in the PostHog destination settings in Segment.
-
-> note ""
-> **Note**: If you're hosting your own PostHog instance, add the URL of your instance without the trailing slash in the **PostHog instance** setting. For example, `https://posthog-example.herokuapp.com`
+6. Enter your PostHog instance URL, as the address to your instance **without any trailing slash**, for example:
+  - `https://app.posthog.com` if you are using PostHog Cloud US
+  - `https://eu.posthog.com` if you are using PostHog Cloud EU
+  - `https://posthog-example.herokuapp.com` if you are self-hosting on Heruku
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:22]:
```suggestion
  - `https://posthog-example.herokuapp.com` if you're self-hosting on Heroku
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 2]
```diff
@@ -16,9 +16,10 @@ This destination is maintained by PostHog. For any issues with the destination,
 3. Choose which Source should send data to the PostHog destination.
 4. Go to your [PostHog set up page](https://app.posthog.com/setup), and find and copy the **API key**.
 5. Enter the PostHog API Key that you copied in the PostHog destination settings in Segment.
-
-> note ""
-> **Note**: If you're hosting your own PostHog instance, add the URL of your instance without the trailing slash in the **PostHog instance** setting. For example, `https://posthog-example.herokuapp.com`
+6. Enter your PostHog instance URL, as the address to your instance **without any trailing slash**, for example:
+  - `https://app.posthog.com` if you are using PostHog Cloud US
+  - `https://eu.posthog.com` if you are using PostHog Cloud EU
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:21]:
```suggestion
  - `https://eu.posthog.com` if you're using PostHog Cloud EU
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 3]
```diff
@@ -16,9 +16,10 @@ This destination is maintained by PostHog. For any issues with the destination,
 3. Choose which Source should send data to the PostHog destination.
 4. Go to your [PostHog set up page](https://app.posthog.com/setup), and find and copy the **API key**.
 5. Enter the PostHog API Key that you copied in the PostHog destination settings in Segment.
-
-> note ""
-> **Note**: If you're hosting your own PostHog instance, add the URL of your instance without the trailing slash in the **PostHog instance** setting. For example, `https://posthog-example.herokuapp.com`
+6. Enter your PostHog instance URL, as the address to your instance **without any trailing slash**, for example:
+  - `https://app.posthog.com` if you are using PostHog Cloud US
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:20]:
```suggestion
  - `https://app.posthog.com` if you're using PostHog Cloud US
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 4]
```diff
@@ -5,7 +5,7 @@ id: 5ece242d61055a0b1bb2e103
 ---
 [PostHog](https://posthog.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is self-hosted, open-source analytics product. Get the same powerful features as other product analytics software but keep full control over your data.
 
-This destination is maintained by PostHog. For any issues with the destination, [contact the PostHog Support team](mailto:hey@posthog.com).
+This destination is maintained by PostHog. For any issues with the destination, checkout the [PostHog users slack](https://posthog.com/slack) or [contact the PostHog Support team](mailto:hey@posthog.com).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:8]:
```suggestion
This destination is maintained by PostHog. For any issues with the destination, check out the [PostHog users slack](https://posthog.com/slack) or [contact the PostHog Support team](mailto:hey@posthog.com).
```

**Review by stayseesong — CHANGES_REQUESTED** [review]

File: src/connections/destinations/catalog/posthog/index.md [hunk 5]
```diff
@@ -5,7 +5,7 @@ id: 5ece242d61055a0b1bb2e103
 ---
 [PostHog](https://posthog.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is self-hosted, open-source analytics product. Get the same powerful features as other product analytics software but keep full control over your data.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:6]:
```suggestion
[PostHog](https://posthog.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is self-hosted, open-source analytics product. Get the same powerful features as other product analytics software but keep full control over your data.
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 6]
```diff
@@ -5,7 +5,7 @@ id: 5ece242d61055a0b1bb2e103
...
 
-This destination is maintained by PostHog. For any issues with the destination, [contact the PostHog Support team](mailto:hey@posthog.com).
+This destination is maintained by PostHog. For any issues with the destination, check out the [PostHog users slack](https://posthog.com/slack) or [contact the PostHog Support team](mailto:hey@posthog.com).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:8]:
```suggestion
This destination is maintained by PostHog. For any issues with the destination, check out the [PostHog users slack](https://posthog.com/slack){:target="_blank"} or [contact the PostHog Support team](mailto:hey@posthog.com).
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 7]
```diff
@@ -16,9 +16,10 @@ This destination is maintained by PostHog. For any issues with the destination,
 3. Choose which Source should send data to the PostHog destination.
 4. Go to your [PostHog set up page](https://app.posthog.com/setup), and find and copy the **API key**.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:17]:
```suggestion
4. Go to your [PostHog set up page](https://app.posthog.com/setup){:target="_blank"}, and find and copy the **API key**.
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 8]
```diff
@@ -3,9 +3,9 @@ title: PostHog Destination
 rewrite: true
 id: 5ece242d61055a0b1bb2e103
 ---
-[PostHog](https://posthog.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) is self-hosted, open-source analytics product. Get the same powerful features as other product analytics software but keep full control over your data.
+[PostHog](https://posthog.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is self-hosted, open-source analytics product. Get the same powerful features as other product analytics software but keep full control over your data.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:6]:
```suggestion
[PostHog](https://posthog.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank"} is a self-hosted, open-source analytics product. Get the same powerful features as other product analytics software but keep full control over your data.
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 9]
```diff
@@ -14,11 +14,12 @@ This destination is maintained by PostHog. For any issues with the destination,
 1. From the Destinations catalog page in the Segment App, click **Add Destination**.
 2. Search for "PostHog" in the Destinations Catalog, and select the PostHog destination.
 3. Choose which Source should send data to the PostHog destination.
-4. Go to your [PostHog set up page](https://app.posthog.com/setup), and find and copy the **API key**.
+4. Go to your [PostHog set up page](https://app.posthog.com/setup){:target="_blank"}, and find and copy the **API key**.
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:17]:
```suggestion
4. Go to your [PostHog set up page](https://app.posthog.com/setup){:target="_blank"}, and copy the **API key**.
```

File: src/connections/destinations/catalog/posthog/index.md [hunk 10]
```diff
@@ -14,11 +14,12 @@ This destination is maintained by PostHog. For any issues with the destination,
...
 5. Enter the PostHog API Key that you copied in the PostHog destination settings in Segment.
-
-> note ""
-> **Note**: If you're hosting your own PostHog instance, add the URL of your instance without the trailing slash in the **PostHog instance** setting. For example, `https://posthog-example.herokuapp.com`
+6. Enter your PostHog instance URL, as the address to your instance **without any trailing slash**, for example:
```
Comment (stayseesong) [inline src/connections/destinations/catalog/posthog/index.md:19]:
```suggestion
6. Enter your PostHog instance URL as the address to your instance **without any trailing slash**, for example:
```

**Review by stayseesong — APPROVED** [review]

**Comment by xvello** [comment]
Thanks a lot @stayseesong!

**Pull request merged**